src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+    public static String Underscores() {
+        return "__________________________________________";
+    }
+
+    public static void help() {
+        String instructions = "The following is a list of commands that you can enter:\n"
+                + "1) help - For showing all commands available\n"
+                + "2) list - To display all the tasks in your list\n"
+                + "3) todo - Add a to-do task into your list (Example: todo borrow book)\n"
+                + "4) deadline - Use \" /by \" to denote deadline (Example: deadline return book /by Sunday)\n"
+                + "5) event - Use \" /at \" to denote when the event takes place (Example: event meeting /at Mon 2-4pm)\n";
+        System.out.println(instructions);
+    }
+
+    public static void greet() {
+        System.out.println(Underscores() + "\nHello! I'm Jave" + "\nWhat can I do for you?");
+    }
+
+    public static void farewell() {
+        System.out.println(Underscores() + "\nBye. Hope to see you again soon!\n" + Underscores());
+    }
+
+    public static void listPrinter(ArrayList<Task> items) {
+        int index = 1;
+        for (Task item : items) {
+            System.out.println(index + ". " + (item).taskPrinter());
+            index++;
+        }
+    }
+
```

Comment:
Overall, the code is readable and mostly follows the coding standards. 

However, the method names, while still somewhat reflecting the methods' purposes, deviate slightly from the coding standards prescribed. For example, the method farewell, is a single noun. Perhaps it could be better renamed printFarewellMessage. Another example is listPrinter, which is not a verb, but a noun. Perhaps it could be better named printList. 

Still, there are many other instances where the naming is properly done, such as in markDone and addedTask, and so these slight errors might be due to carelessness